frappe/utils/__init__.py

```diff
@@ -217,7 +217,7 @@ def split_emails(txt):
 	# emails can be separated by comma or newline
 	s = WHITESPACE_PATTERN.sub(" ", cstr(txt))
 	for email in MULTI_EMAIL_STRING_PATTERN.split(s):
-		email = strip(cstr(email))
+		email = strip(cstr(email), chars="'\"").strip()
```

Comment:
should we really handle these?

also, emails can have single quotes in between them

https://stackoverflow.com/a/4816466

Reply:
the quotes around emails seems to be a msft quirk (yet again): https://www.reddit.com/r/Outlook/comments/pnckao/comment/hcotqfw/?utm_source=share&utm_medium=web2x&context=3

Reply:
Quotes in them are fine, I doubt quotes around email are supported?

Reply:
from https://en.wikipedia.org/wiki/Email_address#Local-part

> If quoted, it may contain Space, Horizontal Tab (HT), any ASCII graphic except Backslash and Quote and a quoted-pair consisting of a Backslash followed by HT, Space or any ASCII graphic; it may also be split between lines anywhere that HT or Space appears. In contrast to unquoted local-parts, the addresses ".John.Doe"@example.com, "John.Doe."@example.com and "John..Doe"@example.com are allowed.

Reply:
reverted, anyways gmail seems to be rejecting this (but will be fine after 3 failed attempts)

Reply:
we could do this if we want: check from right if quotes are present as domain isn't allowed to have them